network/peer/metrics.go

```diff
@@ -107,6 +108,14 @@ func NewMetrics(
 	for _, op := range message.ExternalOps {
 		m.MessageMetrics[op] = NewMessageMetrics(op, namespace, registerer, &errs)
 	}
+
+	m.ClockSkew = metric.NewAveragerWithErrs(
+		namespace,
+		"clock_skew",
+		fmt.Sprintf("clock skew during peer handshake", message.VersionOp.String()),
```

Comment:
```suggestion
		"clock skew during peer handshake",
```
If we don't want the `VersionOp` in the string anymore - we shouldn't have the `Sprintf`